modules/installation-configuration-parameters.adoc

```diff
@@ -510,13 +539,39 @@ ifdef::ibm-z[]
 |String
 endif::ibm-z[]
 
-ifdef::ibm-power[]
+ifdef::ibm-power,ibm-power-vs[]
 |`compute.architecture`
 |Determines the instruction set architecture of the machines in the pool. Currently, heteregeneous clusters are not supported, so all pools must specify the same architecture. Valid values are `ppc64le` (the default).
 |String
-endif::ibm-power[]
+endif::ibm-power,ibm-power-vs[]
 endif::openshift-origin[]
 
+ifdef::ibm-power-vs[]
+|`platform.powervs.vpcRegion`
+|Specifies the IBM Cloud region in which to create VPC resources.
+|String. For example `existing_vpc_region`.
+
+|`platform.powervs.vpc`
```

Comment:
we should remove VPC, Subnets, and pvsNetworkName from this. sorry i didn't catch that sooner

Reply:
Removed : platform.powervs.vpc  , platform.powervs.subnets and platform.powervs.pvsNetworkName from the module.